app/services/book_generator_service.rb

```diff
@@ -0,0 +1,226 @@
+# frozen_string_literal: true
+
+# Usage in console
+# load Rails.root.join('app/services/book_generator_service.rb')
+
+module BookGeneratorService
+  include Rails.application.routes.url_helpers
+
+  def self.generate_book(recipes)
+    template = Rails.public_path.join('fh_cover.jpeg')
+
+    pdf_options = {
+      page_size: 'A4', page_layout: :landscape,
+      left_margin: 0, right_margin: 0,
+      top_margin: 50, bottom_margin: 20, skip_encoding: true
+    }
+
+    pdf = Prawn::Document.new(pdf_options)
+    pdf.state.store.root.data[:Names] = 'Test'
+    pdf.canvas do
+      pdf.image(template, scale: 0.3, at: pdf.bounds.top_left)
+    end
+
+    pdf.font_families.update('Futura' => {
+                               bold: Rails.public_path
+                               .join('futura_medium.ttf'),
+                               medium: Rails.public_path
+                               .join('futura_medium.ttf'),
+                               normal: Rails.public_path
+                               .join('futura_light.ttf')
+                             })
+
+    pdf.font_families.update('AbrilFatface' => {
+                               normal: Rails.public_path
+                               .join('abril_fatface_regular.ttf')
```

Comment:
They also changed the font to: https://fonts.google.com/specimen/Raleway for the header (FOODHUB) and "Social Cooking", could we update to that? :)